src/Compilers/CSharp/Test/Semantic/Semantics/DynamicTests.cs

```diff
@@ -3434,6 +3434,33 @@ static void Main()
             CompileAndVerify(compilation2, expectedOutput: @"4");
         }
 
+        [Fact, WorkItem(9945, "https://github.com/dotnet/roslyn/issues/9945")]
+        public void DynamicGetOnlyProperty()
```

Comment:
Two places in binding were modified, but it looks like this test covers only one of them.